Assets/Mirror/Tests/Editor/NetworkIdentityTests.cs

```diff
@@ -12,22 +12,21 @@ namespace Mirror.Tests
 {
     public class NetworkIdentityTests
     {
-        #region test components
+       #region test components
         class MyTestComponent : NetworkBehaviour
```

Comment:
Not for this PR,  but all of these test components can now be removed and replaced by mocks

Reply:
Agree that alot of this needs another pass in the future as changes in logic are made.